src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Credential/CredentialHandler.cs

```diff
@@ -0,0 +1,94 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.SqlServer.Management.Smo;
+using Microsoft.SqlTools.ServiceLayer.Connection;
+using Microsoft.SqlTools.ServiceLayer.Management;
+
+namespace Microsoft.SqlTools.ServiceLayer.ObjectManagement
+{
+    /// <summary>
+    /// Credential object type handler
+    /// </summary>
+    public class CredentialHandler : ObjectTypeHandler
+    {
+        public CredentialHandler(ConnectionService connectionService) : base(connectionService)
```

Comment:
This class was built a while back to a different protocol as user/login, so not sure if it will work with updates, etc.  We can consider deleting if we aren't going to build UI for this object type in the near future.

Reply:
for credential object type, I only implemented the create/drop operations that are needed for the integration tests.